api/v1alpha1/proxmoxcluster_types_test.go

```diff
@@ -106,6 +107,16 @@ var _ = Describe("ProxmoxCluster Test", func() {
 		Expect(client.IgnoreNotFound(err)).To(Succeed())
 	})
 
+	Context("ClusterPort", func() {
+		It("Should not allow invalid ports", func() {
+			dc := defaultCluster()
+			dc.Spec.ControlPlaneEndpoint = &clusterv1.APIEndpoint{
+				Port: 65536,
```

Comment:
might want to do a test for `0` as that's commonly used as a placeholder/don't-care/default value/omitted.